app/views/vm_common/_disassociate_floating_ip.html.haml

```diff
@@ -8,12 +8,13 @@
         = _('Floating IP')
       .col-md-8
         %select{:name      => 'floating_ip',
-                'ng-model' => 'vmCloudModel.floating_ip',
-                'ng-options' => 'floating_ip.address for floating_ip in floating_ips track by floating_ip.address'}
+                'ng-model' => 'vm.vmCloudModel.floating_ip',
+                'ng-options' => 'floating_ip.address for floating_ip in vm.floating_ips track by floating_ip.address'}
+
+    %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
+                    'paging_div_buttons_id'            => "angular_paging_div_buttons",
+                    'paging_div_buttons_type'          => "Submit"}
 
-  %div_for_paging{'ng-controller'                    => "pagingDivButtonGroupController",
-                  'paging_div_buttons_id'            => "angular_paging_div_buttons",
-                  'paging_div_buttons_type'          => "Submit"}
 
 - unless @explorer
```

Comment:
This is definitely broken .. looks like you had to move this right in `associate`, so that the buttons are part of the form .. but forgot to do the same here?